src/book-byen/book-byen-admin.json

```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "key": "book-byen-form-1",
+    "input": "header",
+    "text": "Skabelon: Book Byen",
+    "name": "header1",
+    "formGroupClasses": "h4 mb-3"
+  },
+  {
+    "key": "book-byen-form-2",
+    "input": "header-h3",
+    "text": "Indhold",
+    "name": "header2",
+    "formGroupClasses": "h5 mb-3"
+  },
+  {
+    "key": "book-byen-form-3",
+    "input": "textarea",
+    "name": "title",
+    "label": "Overskrift på slide",
+    "helpText": "Her kan du skrive overskriften til slidet",
```

Comment:
There are a few entries here: `text`, `name`, `label` and `helpText`. The usage, language and "quality" of the content of each of these entries all through the file vary quite a bit.

Please see if you can streamline these.

Reply:
I'll handle this in a follow up when I know what should actually happen